Rakefile_appveyor_test

```diff
@@ -0,0 +1,31 @@
+# -*- ruby -*-
+
+require 'rubygems'
+require 'rubygems/package_task'
+require "rake/testtask"
+
+begin
+  require 'psych'
+rescue ::LoadError
+  require 'yaml'
+end
+
+# Stupid hack to copy minitest files into repo folder
+if RUBY_VERSION <= '1.9.3'
+  dest = File.dirname(__FILE__)
+  src_dir = Dir.glob("#{Gem.default_dir}/gems/minitest-*").sort
+  if src_dir.last
+    cmd = "copy #{src_dir.last}/lib/minitest.rb #{dest}/lib".gsub(/\//, "\\")
```

Comment:
can we use the Rake `cp` command here? And if not, can we use `sh` instead of backticks? We probably should also wrap this in a command, that the `test` task depends upon

Reply:
I guess we could.  At present, it's windows only, and it's only required for 1.9.3.  All the other version pass without it.